src/main/utils.js

```diff
@@ -49,8 +50,7 @@ const create_store = function ({
 
 const get_flask_logs_full_path = function (electron_store) {
   const electron_store_dir = path.dirname(electron_store.path);
-  const flask_logs_subfolder = "logs_flask";
-  return path.join(electron_store_dir, flask_logs_subfolder);
+  return path.join(electron_store_dir, "logs_flask", log_uuid);
```

Comment:
this is not be very user friendly for when they need to access the logs, they'll have to really pay attention to what the model says in order to get the correct uuid for that sessions logs. Think this be changed to `mantarray_logs__Y_M_D_Time` of the main process log instead of a uuid?

Reply:
Yeah that's a good point, if we needed them to send us logs it would be tough to explain which ones to send. I like the `mantarray_logs_YYMMDD_Time` format